### PR TITLE
fix(fossid-webapp): Generate ignore rules also for non-delta scans

### PR DESCRIPTION
**Pull request by wkl3nk**
Generate ignore rules from path excludes in _.ort.yml_ also if FossID delta scans are disabled.

### CODE REVIEW

File: plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt [hunk 1]
```diff
@@ -935,6 +935,45 @@ class FossIdTest : WordSpec({
             }
         }
 
+        "apply exclusion rules to a origin scan" {
+            val projectCode = projectCode(PROJECT)
+            val scanCode = scanCode(PROJECT, FossId.DeltaTag.ORIGIN)
+            val config = createConfig()
```
Comment (nnobelis) [inline plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt:941]: Your PR is adding the logic to convert rules in the case when this is NOT a delta scope.
Therefore you need a `createConfig(deltaScans= false)`.
You can test with the debugger that you are in the right branch of the code.

File: plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt [hunk 2]
```diff
@@ -935,6 +935,45 @@ class FossIdTest : WordSpec({
             }
         }
 
+        "apply exclusion rules to a origin scan" {
+            val projectCode = projectCode(PROJECT)
+            val scanCode = scanCode(PROJECT, FossId.DeltaTag.ORIGIN)
```
Comment (nnobelis) [inline plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt:940]: You need a scancode without the delta part (no really critical but consistency).

File: plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt [hunk 3]
```diff
@@ -935,6 +935,45 @@ class FossIdTest : WordSpec({
             }
         }
 
+        "apply exclusion rules to a origin scan" {
```
Comment (nnobelis) [inline plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt:938]: Origin scans are only for delta scans.

How about "apply exclusion rules to a non-delta scan"

File: plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt (hunk 1 above)
Comment (wkl3nk) [reply plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt]: Thanks for the hint. Using the debugger I am now sure that the desired code is covered.

File: plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt (hunk 2 above)
Comment (wkl3nk) [reply plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt]: Had a look at other non-delta scans and modified the code in the same way.

**Review by nnobelis — APPROVED** [review]

**Review by nnobelis — APPROVED** [review]

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**